### PR TITLE
ADCM-3057 Add stdout logger

### PR DESCRIPTION
**Pull request by winmasta**

### CODE REVIEW

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

File: os/etc/service/wsgi/run [hunk 1]
```diff
@@ -16,8 +16,5 @@ source /etc/adcmenv
 waitforinit
 
 echo "Run main wsgi application ..."
-exec 1>>"${adcmlog}/service_wsgi.out"
-exec 2>>"${adcmlog}/service_wsgi.err"
-
 cd "${adcmroot}/python"
-uwsgi --venv "${adcmroot}/venv/default" --socket "${wsgisocketfile}" --pidfile "/run/uwsgi.pid" --module adcm.wsgi --chmod-socket=777 --logger file:logfile="${adcmlog}/wsgi.log",maxsize=2000000
+uwsgi --venv "${adcmroot}/venv/default" --socket "${wsgisocketfile}" --pidfile "/run/uwsgi.pid" --module adcm.wsgi --chmod-socket=777
```
Comment (a-alferov) [inline os/etc/service/wsgi/run:20]:
```suggestion
uwsgi --venv "${adcmroot}/venv/default" --socket "${wsgisocketfile}" --pidfile "/run/uwsgi.pid" --module adcm.wsgi --chmod-socket=777 --disable-logging
```
Add pls `--disable-logging` for disable UWSGI logs and show only Django logs

**Review by a-alferov — CHANGES_REQUESTED** [review]

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Review by a-alferov — APPROVED** [review]

**Pull request merged**